afformEntities/ContactNamePair.php

```diff
@@ -1,6 +1,4 @@
 <?php
 return [
-  'entity' => 'ContactNamePair',
```

Comment:
@colemanw is this not being required civi-version dependent? Or was it never required just copy & paste acquired?

Reply:
It depends on https://github.com/civicrm/civicrm-core/pull/20283

Reply:
If you kept the `'entity' => 'ContactNamePair'` line then the rest of this PR would work with previous versions of Afform.

Reply:
Hmm - maybe I'll just wait to cut it until I'm ready to make it 5.39 dependent - otherwise I'l forget this cleanup